google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentRunConfigurationEditor.java

```diff
@@ -386,10 +387,17 @@ private void validateConfiguration() throws ConfigurationException {
         throw new ConfigurationException(
             GctBundle.message("appengine.flex.config.custom.dockerfile.error"));
       }
-    } else if (environment.isStandard() && CloudSdkService.getInstance().validateCloudSdk()
-        .contains(CloudSdkValidationResult.NO_APP_ENGINE_COMPONENT)) {
-      throw new ConfigurationException(
-          CloudSdkValidationResult.NO_APP_ENGINE_COMPONENT.getMessage());
+    } else if (environment.isStandard()) {
+      Set<CloudSdkValidationResult> validationResults =
+          CloudSdkService.getInstance().validateCloudSdk();
+      if (validationResults.contains(CloudSdkValidationResult.CLOUD_SDK_NOT_FOUND)) {
```

Comment:
Also, shouldn't this be checked regadless of whether the environment is standard or flex?